be/CMakeLists.txt

```diff
@@ -555,23 +556,24 @@ if (${MAKE_TEST} STREQUAL "ON")
     add_definitions(-DBE_TEST)
 endif ()
 
+add_subdirectory(${SRC_DIR}/agent)
 add_subdirectory(${SRC_DIR}/codegen)
 add_subdirectory(${SRC_DIR}/common)
-add_subdirectory(${SRC_DIR}/util)
+add_subdirectory(${SRC_DIR}/env)
```

Comment:
why here to modifly the positions of the sub directories?

Reply:
Just sort by name